.github/workflows/test-macos.yml

```
@@ -25,6 +25,6 @@ jobs:
       - name: Environment Information
         run: npx envinfo
       - name: Build
-        run: make build-ci -j8 V=1 CONFIG_FLAGS="--error-on-warn --experimental-quic"
+        run: make build-ci -j2 V=1 CONFIG_FLAGS="--error-on-warn --experimental-quic"
```

Why not let `make` decide? If GitHub were to change their policy in the future, the action would pick it up automatically.

> With no argument, make runs as many recipes simultaneously as possible. [[Source](https://www.gnu.org/software/make/manual/make.html#Options-Summary)]

```suggestion
        run: make build-ci -j V=1 CONFIG_FLAGS="--error-on-warn --experimental-quic"
```

---

all the other actions are explicit about number of cores. I feel like this would be a good follow up doing this for all actions
thoughts?